internal/resources/credential/resource_credential.go

```diff
@@ -43,6 +44,12 @@ var credentialSchema = map[string]*schema.Schema{
 		Computed:    true,
 		Elem:        &schema.Schema{Type: schema.TypeString},
 	},
+	waitKey: {
+		Type:        schema.TypeString,
+		Description: "Wait timeout duration until credential resource reaches VALID state. Accepted timeout duration values like 5s, 5m, or 1h, higher than zero.",
+		Default:     "default",
```

Comment:
Default wait time is "default"? why can't we set a default time instead

Reply:
For having a consistent user experience as existing are using "default",

Reply:
Existing as in for cluster resource? 
I see in the switch case where we set a default time, cannot we set it here?